.composer.json

```diff
@@ -14,7 +14,7 @@
   "scripts": {
     "lint:phpcs-psr12": "../../bin/phpcs --colors --standard=PSR12 ./Neos.ContentGraph.DoctrineDbalAdapter/src ./Neos.ContentGraph.PostgreSQLAdapter/src ./Neos.ContentRepository.BehavioralTests/Classes ./Neos.ContentRepository.TestSuite/Classes ./Neos.ContentRepository.Core/Classes ./Neos.Neos/Classes",
     "lint:phpcs": [
-      "@lint:phpcs-psr12 --exclude=Generic.Files.LineLength"
+      "@lint:phpcs-psr12 --exclude=Generic.Files.LineLength,PSR1.Files.SideEffects"
```

Comment:
Why do we need to change the linter rules for this change? that makes me suspicious :)

Reply:
This was a problem with phpcs and `readonly` - but it should have been fixed with: https://github.com/squizlabs/PHP_CodeSniffer/issues/3727

Reply:
but they are stuck a bit in their release workflow. We have to wait patiently ;)

Excluding `PSR1.Files.SideEffects` is actually no big deal as we write proper php code generally ^^

```
FILE: /Neos.ContentRepository.Core/Classes/Projection/ContentGraph/Node.php
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 1 | WARNING | A file should declare new symbols (classes, functions, constants, etc.) and cause no other side effects, or it should execute logic with side effects, but should not do both. The first symbol is defined on line 37 and the first side effect is on line 37.
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```